Sources/Sentry/SentryAppStateManager.m

```diff
@@ -85,43 +84,48 @@ - (void)start
 }
 
 - (void)stop
+{
+    [self stop:NO];
+}
+
+- (void)stop:(BOOL)forceStop
 {
     if (self.startCount <= 0) {
         return;
     }
 
-    self.startCount -= 1;
+    if (forceStop) {
+        self.startCount = 0;
+    } else {
+        self.startCount -= 1;
```

Comment:
So if `forceStop` is not `YES`, then we require developers to correctly balance calls to start/stop? Why can't we just always reset `startCount` to 0?

Reply:
There are 2 integrations that start this manager. And only if both are stopped, should this manager be stopped as well. The only exception is if the entire SDK is closed, (which doesn't call the stop method on every integration). So then we force this to be stopped.

But yeah normally they have to be balanced.